packages/hooks/package.json

```
@@ -33,5 +32,8 @@
     "@nft/api-graphql": "^1.0.0-beta.7",
     "@types/react": "^17.0.51",
     "eslint-plugin-react-hooks": "^4.2.0"
+  },
+  "peerDependencies": {
+    "react": ">=18"
```

Let's keep it compatible with react 17 as well so it doesn't break all the other apps
```suggestion
    "react": ">=17.0.0"
```

---

[Change min react to 17](https://github.com/liteflow-labs/liteflow-js/pull/97/commits/ab23ebad10f5097c370876c845a03af778b81c82)